sc-memory/sc-memory/scs/scs_types.cpp

```diff
@@ -22,30 +22,37 @@ TypeResolver::MapType TypeResolver::ms_connectorToType = {
     {"<=", ScType::EdgeDCommonConst},
     {"_=>", ScType::EdgeDCommonVar},
     {"_<=", ScType::EdgeDCommonVar},
+    {"<=_", ScType::EdgeDCommonVar},
```

Comment:
You should to to remove this features. They broke scs-texts handling

Reply:
Add tests, it works correctly